### PR TITLE
Wiki  update

### PR DESCRIPTION
**Pull request by alsmith1**

### CODE REVIEW

File: wikiwiki/Case-Information.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+* Petitioners Counsel information listed
+* Add council functionality to search for and add a respondent council
+* Edit functionality available to internal court users
+* Petitioner's Council Mapping: [name][barnumber] (RT6789) [address1] [address2] [address3] [city], [state] [postalCode] [phone number]
 Service email [Service Preference] Representing [petitioner]
+
+**Other Petitioners**
+* Petitioner Contact tile size spans full width when no spouse contact available
+displays under the main petitioner contact tile
```
Comment (matthopson) [inline wikiwiki/Case-Information.md]: Is this dangling on purpose?

File: wikiwiki/Consolidated-Cases.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+[Return to Directory](./README.md)
+
+## Consolidated Cases
+
+* Cases are consolidated to save time and money in court proceedings
+* Consolidated cases are scheduled together for court trials
+* Petitioner or practitioner needs to submit a document to the court in order to be considered for consolidating cases.
```
Comment (matthopson) [inline wikiwiki/Consolidated-Cases.md]: Does it matter that this bullet has a period?

File: wikiwiki/Consolidated-Cases.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+**Once cases have been consolidated:**
+* A yellow lock double paper icon will display in the case header
+* The consolidated cases tile on case information tab will display all of the cases consolidated (viewable to internal court users only)
+* Consolidated cases are listed in ascending order with the lowest docket number the lead case
+* Petitioner/Practitioner View of Consolidated Cases
+
+**On a petitioner/practitioner dashboard users will see:**
+* Consolidated cases on the open/closed case tabs display a yellow lock double paper icon next to the lead case
+* The lead case has the lowest docket number. the rest of the cases will display indented below the lead case in ascending order.
```
Comment (matthopson) [inline wikiwiki/Consolidated-Cases.md]: `the` after `.` needs capitalization. Should it end with a period (the rest don't)?

**Review by matthopson — APPROVED** [review]
Just a couple nits.

**Comment by alsmith1** [comment]
@matthopson updates have been made. thanks for the review

**Pull request merged**